Cargo.toml

```diff
@@ -19,8 +19,12 @@ license = "MIT"
 edition = "2021"
 exclude = [".dir-locals.el"]
 
+[features]
+default = ["getrandom"]
+getrandom = ["rand/getrandom", "rand/std_rng", "rand/std"]
```

Comment:
About the name of the feature: as I was reading through the PR, I was thinking that `thread_rng` might be better since that is actually what the feature controls: the availability of `rand::thread_rng` (assuming I read things correctly).

Does that sound right?

Reply:
Obsolete now that we don't need `thread_rng` any longer.